configure.ac

```diff
@@ -1318,19 +1331,27 @@ AC_MSG_NOTICE([-------------------------])
 AC_MSG_NOTICE([  Configuration summary  ])
 AC_MSG_NOTICE([-------------------------])
 
-AC_MSG_NOTICE([  PETSc support           : $HAS_PETSC (has SUNDIALS: $PETSC_HAS_SUNDIALS)])
-AC_MSG_NOTICE([  SLEPc support           : $HAS_SLEPC])
-AC_MSG_NOTICE([  IDA support             : $HAS_IDA])
-AC_MSG_NOTICE([  CVODE support           : $HAS_CVODE])
-AC_MSG_NOTICE([  ARKODE support          : $HAS_ARKODE])
-AC_MSG_NOTICE([  FFTW support            : $HAS_FFTW])
-AC_MSG_NOTICE([  NetCDF support          : $HAS_NETCDF])
-AC_MSG_NOTICE([  Parallel-NetCDF support : $HAS_PNETCDF])
-AC_MSG_NOTICE([  HDF5 support            : $HAS_HDF5 (parallel: $HAS_PHDF5)])
-AC_MSG_NOTICE([  Lapack support          : $HAS_LAPACK])
-AC_MSG_NOTICE([  Scorep support          : $HAS_SCOREP])
-AC_MSG_NOTICE([  OpenMP support          : $HAS_OPENMP (schedule: $OPENMP_SCHEDULE)])
-AC_MSG_NOTICE([  Natural language support: $HAS_NLS (path: $localedir)])
+AC_MSG_NOTICE([  PETSc support           : $BOUT_HAS_PETSC (has SUNDIALS: $PETSC_HAS_SUNDIALS)])
+AC_MSG_NOTICE([  SLEPc support           : $BOUT_HAS_SLEPC])
+AC_MSG_NOTICE([  IDA support             : $BOUT_HAS_IDA])
+AC_MSG_NOTICE([  CVODE support           : $BOUT_HAS_CVODE])
+AC_MSG_NOTICE([  ARKODE support          : $BOUT_HAS_ARKODE])
+AC_MSG_NOTICE([  FFTW support            : $BOUT_HAS_FFTW])
+AC_MSG_NOTICE([  NetCDF support          : $BOUT_HAS_NETCDF])
+AC_MSG_NOTICE([  Parallel-NetCDF support : $BOUT_HAS_PNETCDF])
+AC_MSG_NOTICE([  HDF5 support            : $BOUT_HAS_HDF5 (parallel: $BOUT_HAS_PHDF5)])
```

Comment:
Are the parallel interfaces working?
I never managed to get them working, so might be better to remove them?

Reply:
They don't work, but I think they're about 90% implemented. If anyone ever does have the motivation and time to finish them, I think the existing code's not necessarily useless. If we haven't documented somewhere that they don't work, then we should do that...

Reply:
One of the blockers for v5, as far as I'm concerned at least, is refactoring `Datafile`, and that will include dealing with them one way or another!